gitignore.example

```diff
@@ -2,6 +2,7 @@
 
 # Temporary build-tools artifacts to be ignored
 /.go/
```

Comment:
I couldn't find any mention in the dep repo of whether it is or not. I'd say merge away if you are comfortable. It's a pretty easy adjustment if we happen to get it wrong.